aiida/cmdline/commands/cmd_setup.py

```diff
@@ -177,8 +178,8 @@ def quicksetup(
                 'Oops! quicksetup was unable to create the AiiDA database for you.',
                 'See `verdi quicksetup -h` for how to specify non-standard parameters for the postgresql connection.\n'
                 'Alternatively, create the AiiDA database yourself: ',
-                manual_setup_instructions(dbuser=su_db_username,
-                                          dbname=su_db_name), '', 'and then use `verdi setup` instead', ''
+                manual_setup_instructions(db_username=db_username,
```

Comment:
Not concerning this line, but line 184: should we just reraise the exception? Isn't it better to simply use `echo.echo_critical`? Usually showing a scary traceback by default to a user from a CLI command is not a good idea.

Reply:
Also just noted that the instructions above (to use `verdi quicksetup -h` for help) were wrong for a long time and only (coincidentally) got fixed just two weeks ago in 6469e2360680ff309c2275dbe205b979e2987ac6  :smile:

Reply:
Sometimes the traceback can also help to see where the problem is... let me know if you want me to change it

Reply:
There is a reusable option that we have `--traceback` that will optionally show the traceback which I think can be useful. But we can leave it for now, it is not critical